### ISSUE TITLE
pushbutton default footprint don't work anymore; regression

### ISSUE THREAD

**Issue by ShiboSoftwareDev** [tscircuit/footprinter#88]
![Image](https://github.com/user-attachments/assets/ef6165cb-8aad-49c6-854b-fcffd3a13a39)

**Comment by AnasSarkiz** [comment]
@seveibar 
We have changed the pushbutton parameter zod type to `literal` in #83. You need to pass the `pushbutton` string


![Image](https://github.com/user-attachments/assets/ae4df166-cd3d-4c88-824c-e4b452a701a8)


this is a test file

![Image](https://github.com/user-attachments/assets/5bfa1bb7-5ae0-4380-a51e-3c74f3c43531)

let me know if there are any changes needed
cc: @seveibar

**Comment by seveibar** [comment]
I don't understand, shibo gave a valid footprinter string- what is the valid string if not that?

**Comment by seveibar** [comment]
Oh are you saying that the parameters are just not customizable at all? @AnasSarkiz

**Comment by seveibar** [comment]
@ShiboSoftwareDev can you detect if the zod schema contains a literal woth a default and just not set it in the footprinter string? Sorry for the confusion, i think i understand now (maybe)

**Comment by AnasSarkiz** [comment]
yes, he can customize any of them. the only thing he can do is write  `pushbutton` as footprinter string
